language/stdlib/modules/LibraSystem.move

```diff
@@ -50,21 +46,19 @@ module LibraSystem {
             EINVALID_SINGLETON_ADDRESS
         );
 
-        let cap = LibraConfig::publish_new_config_with_capability<LibraSystem>(
+        LibraConfig::publish_new_config<LibraSystem>(
             config_account,
             LibraSystem {
                 scheme: 0,
                 validators: Vector::empty(),
             },
         );
-        move_to(config_account, CapabilityHolder { cap })
     }
 
-    // This copies the vector of validators into the LibraConfig's resource
-    // under ValidatorSet address
-    fun set_validator_set(value: LibraSystem) acquires CapabilityHolder {
-        LibraConfig::set_with_capability<LibraSystem>(&borrow_global<CapabilityHolder>(CoreAddresses::LIBRA_ROOT_ADDRESS()).cap, value)
+    fun set_validator_set(lr_account: &signer, value: LibraSystem) {
```

Comment:
Now that this is a pure wrapper, we might want to get rid of it and just call `LibraConfig::set` at each call site.